packages/edit-site/src/components/global-styles/preview.js

```diff
@@ -78,8 +89,49 @@ const StylesPreview = ( { label, isFocused, withHoverView } ) => {
 	const [ styles ] = useGlobalStylesOutput();
 	const disableMotion = useReducedMotion();
 	const [ isHovered, setIsHovered ] = useState( false );
-	const [ containerResizeListener, { width } ] = useResizeObserver();
-	const ratio = width ? width / normalizedWidth : 1;
+	const [ containerResizeListener, sizes ] = useResizeObserver();
+	const { width } = sizes || { width: 0, height: 0 };
```

Comment:
Not sure if it matters here since there's no `useMemo` or similar relying on the constant ref, but same as for `THROTTLE_OPTIONS`, could `{ width: 0, height: 0 }` be defined outside of the component as well since other defaults are?

Reply:
Whoops! I forgot to revert this line, this one should've been `const [ containerResizeListener, { width } ] = useResizeObserver();`. Good catch! Reverted that line in https://github.com/WordPress/gutenberg/pull/57090/commits/32a33feba5326338c70de22e32f434f96638c017